CHANGELOG.md

```diff
@@ -57,6 +57,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- `opentelemetry-instrumentation-celery` propagates baggage
```

Comment:
You need to move this under the unreleased release

Reply:
Done :ok_hand: